src/pattern-library/components/PlaygroundApp.tsx

```diff
@@ -164,77 +151,88 @@ export default function PlaygroundApp({
   >;
   return (
     <Router base={baseURL}>
-      <main className="max-w-[1250px] mx-auto">
-        <div className="md:grid md:grid-cols-[2fr_5fr]">
-          <div className="md:h-screen md:sticky md:top-0 overflow-scroll">
-            <div className="md:sticky md:top-0 h-16 px-6 flex items-center bg-slate-0 border-b">
-              <h1 className="text-lg">
-                <Link href={baseURL + '/'} classes="grow flex gap-x-2">
-                  <LogoIcon />
-                  Component Library
-                </Link>
-              </h1>
+      <div className="w-full bg-stone-200">
+        <main className="max-w-[1300px] mx-auto">
```

Comment:
A little extra width to accommodate some extra x-padding I put on the main page content.